README.md

```diff
@@ -210,8 +210,20 @@ As of today, ASA does no longer offer a way to query the server, so there's no q
 
 ## Changing the game port and RCON port
 
-You already learned that ports are defined by `ASA_START_PARAMS` in the `docker-compose.yml` file. This just tells the ASA server what ports to bind, but this alone is not enough and you need to apply
-the following changes as well.
+You already learned that ports are defined by `ASA_START_PARAMS` in the `docker-compose.yml` file. This just tells the ASA server what ports to bind.
+As a first step for port changes adjust the start parameters accordingly.
+
+E. g. if you want to change the game port from `7777` to `7755` your new start parameters would be like this:
```

Comment:
`would be like this` -> `would look like this`

Reply:
addressed with [45e3cfc](https://github.com/mschnitzer/ark-survival-ascended-linux-container-image/pull/82/commits/45e3cfc808e52bdf1f1af888a395b8a238ddf458)